views/templates/admin/configuration.tpl

```diff
@@ -40,19 +40,18 @@
    {/if}
    {if isset($gsitemap_refresh_page)}
    <h3><i class="icon icon-sitemap"></i> {l s='Your Sitemaps' d='Modules.Gsitemap.Admin'}</h3>
-   <p>{$gsitemap_number|intval} {l s='Sitemaps were already created.' d='Modules.Gsitemap.Admin'}<br>
-   </p>
-   <br>
+   <p>{$gsitemap_number|intval} {l s='Sitemaps were already created.' d='Modules.Gsitemap.Admin'}</p>
+   <br><br>
    <form action="{$gsitemap_refresh_page|escape:'htmlall':'UTF-8'}" method="post" id="gsitemap_generate_sitmap">
       <img src="../img/loader.gif" alt=""/>
-      <input type="submit" class="button" value="{l s='Continue' d='Modules.Gsitemap.Admin'}" style="display: none;"/>
+      <input type="submit" class="button" value="{l s='Continue' d='Admin.Actions'}" style="display: none;"/>
    </form>
    {else}
    {if $gsitemap_links}
    <h3><i class="icon icon-sitemap"></i> {l s='Your Sitemaps' d='Modules.Gsitemap.Admin'}</h3>
    {l s='Please set up the following Sitemap URL in your Google Webmaster account:' d='Modules.Gsitemap.Admin'}<br> 
    <a href="{$gsitemap_store_url|escape:'htmlall':'UTF-8'}{$shop->id|intval}_index_sitemap.xml" target="_blank">{$gsitemap_store_url|escape:'htmlall':'UTF-8'}{$shop->id|intval}_index_sitemap.xml</a><br><br>
-   {l s='This URL is the master Sitemaps file. It refers to the following sub-sitemap files:' d='Modules.Gsitemap.Admin'}
+   {l s='The above URL is the master sitemap file. It refers to the following sub-sitemap files:' d='Modules.Gsitemap.Admin'}
```

Comment:
Watch out when you change wordings, as they will stop being translated forever on all current versions! Only newer versions (starting on 1.7.5) will have new wordings translated.

Even small changes (a dot, an upper-case letter) will prevent the wording from being translated.

Reply:
I am aware of this, but I think the old text is misleading. The sitemap really needs some changes, and I think we should make all necessary changes now.

Reply:
What do you think @LouiseBonnard @colinegin ?

Reply:
It's always nice to keep improving wordings while fixing; it will kind of duplicate the string thought but I might hide it on Crowdin.